src/AppContainer.re

```diff
@@ -34,7 +33,7 @@ external makeProps:
     ~persistNavigationState: persistNavigationState=?,
     ~loadNavigationState: loadNavigationState=?,
     ~screenProps: 'screenProps=?,
-    ~ref: Js.Nullable.t(NavigationContainer.t) => unit=?,
+    ~ref: option(NavigationContainer.t) => unit=?,
```

Comment:
Not sure that's safe. nullable is for null, option None is encoded as undefined, which is != null.

Reply:
Okay, I'll fix it.